app/controllers/concerns/hydranorth/collections/selects_collections.rb

```diff
@@ -1,5 +1,6 @@
 module Hydranorth::Collections::SelectsCollections
   extend ActiveSupport::Concern
+  # shouldn't this be Sufia::Catalog? We're all over the place with includes
```

Comment:
This was copied directly from Hydra:Collections to overwrite. But I think you are right, should've used Sufia::Catalog instead.